nailgun/entities.py

```diff
@@ -1042,6 +1042,7 @@ def __init__(self, server_config=None, **kwargs):
             'attach_stdin': entity_fields.BooleanField(),
             'attach_stdout': entity_fields.BooleanField(),
             'command': entity_fields.StringField(
+                default='top',
                 required=True,
                 str_type='latin1',
```

Comment:
Tried it out with 'utf8' - same result. But not for 'numeric', for example, it is working ok. So i guess it's working ok with default alphanumeric characters and is failing on special ones.
The thing is - if you pass some random command like '012a4754955fa33cdadd2109fd2f836ae46a15f010d2617ad0356b48b826d614' - creation will still fail with 500 error and following reason:
```
2016-06-23 13:42:08 - nailgun.client - WARNING - Received HTTP 500 response: {
  "error": {"message":"Cannot start container 012a4754955fa33cdadd2109fd2f836ae46a15f010d2617ad0356b48b826d614: [8] System error: exec: \"162817867131753\": executable file not found in $PATH\n"}
}
```
This basically means there's no point in generating random value for this field by default, we should use predefined one in `default=` parameter, as otherwise nothing will be created. Therefore, i'll drop this `str_type=` line.